<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="GroupInput">
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="InputText" name="TextEditInput">
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GroupOutput">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="TextEditOutput">
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLayoutSettings">
         <item>
          <widget class="QPushButton" name="ButtonAbout">
           <property name="text">
            <string>About</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer1">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="LabelWidth">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBoxWidth">
           <property name="toolTip">
            <string>Minimum width of the box, taking in account the indentation. Box is extended if commentaries are wider</string>
           </property>
           <property name="minimum">
            <number>20</number>
           </property>
           <property name="maximum">
            <number>250</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="LabelEmptyLines">
           <property name="text">
            <string>Empty lines:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBoxEmptyLines">
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="LabelIndent">
           <property name="text">
            <string>Indent:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBoxIndent">
           <property name="suffix">
            <string> (tabs)</string>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="LabelTabSize">
           <property name="text">
            <string>Tab size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBoxTabSize">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonSettings">
           <property name="text">
            <string>More settings</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ButtonCopy">
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputText</class>
   <extends>QPlainTextEdit</extends>
   <header>InputText.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
